pycaw/callbacks.py

```diff
@@ -1,3 +1,108 @@
-# TODO
-# implement easy to use callbacks
-# (is already written, but 1 pull at the time)
+from ctypes import pointer
+
+from comtypes import COMObject
+
+from pycaw.api.audiopolicy import (IAudioSessionControl2, IAudioSessionEvents,
+                                   IAudioSessionNotification)
+from pycaw.api.endpointvolume import IAudioEndpointVolumeCallback
+from pycaw.utils import AudioSession
+
+
+class AudioSessionNotification(COMObject):
+    """
+    SESSION CREATED CALLBACK
+
+    """
```

Comment:
I'd make this a single line docstring https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
thanks ;) that would be better ;)